observability-kit-starter/src/main/resources/META-INF/resources/frontend/components/observability-client.ts

```diff
@@ -39,10 +40,22 @@ export class ObservabilityClient extends LitElement {
   protected firstUpdated(_changedProperties: PropertyValues) {
     super.firstUpdated(_changedProperties);
 
+    const resource =
+      Resource.default().merge(
+        new Resource({
+          [SemanticResourceAttributes.SERVICE_NAME]: this.serviceName,
+          [SemanticResourceAttributes.SERVICE_VERSION]: this.serviceVersion,
+        })
+      );
+
+    const provider = new WebTracerProvider({
```

Comment:
Should we perhaps hold a reference to the provider and call its `shutdown()` function in the `disconnectedCallback`? Does it make sense?